app/src/componentLibrary/dropdown/dropdown.jsx

```diff
@@ -66,7 +66,7 @@ export const Dropdown = ({
   };
 
   const getDisplayedValue = () => {
-    if (!value) return placeholder;
+    if (!value && value !== false) return placeholder;
```

Comment:
For what reason we must have this here?
!value covere all situations when the value is falsy value. Is`t it?